### ISSUE TITLE
Filters should be per camera

### ISSUE THREAD

**Issue by gshlomi** [sclaflin/Plate-Minder#9]
Hi,
It looks like the filters configuration is global, so when using multiple sources, one camera's mask might cover required detection region on another camera.
Can you change so the filters configuration be per camera?

Thanks

**Comment by sclaflin** [comment]
You are correct, the filters configuration is global. I think a good approach should be to add camera specific filters in addition to the existing global filters. Currently, my availability is a bit low, but I can add this to the list of future updates. Pull requests are also welcome.

**Comment by Iceman248** [comment]
Is there a way to do camera specific filters yet?

**Comment by sclaflin** [comment]
Hi there, not at this time.

**Comment by Iceman248** [comment]
Thanks. There any other way to have any kind of filter specific to a camera besides running an instance per camera of Plate-Minder?

**Comment by sclaflin** [comment]
Unfortunately, no. It's definitely a planned feature, but I'm just some schlub with very little free time.
